src/index.js

```diff
@@ -31,6 +36,9 @@ Dotdotdot.prototype.dotdotdot = function(container) {
     });
   }
 };
+Dotdotdot.prototype.update = function() {
```

Comment:
It's not needed. It was added only for sake of clean code.
I'll update this in few secs.